nbs/animation.ipynb

```diff
@@ -118,9 +105,11 @@
         "\n",
         "show_documentation = True # @param {type:'boolean'}\n",
         "\n",
-        "###################\n",
+        "# #@markdown ####**Resume:**\n",
+        "resume_timestring = \"\" #@param {type:\"string\"}\n",
```

Comment:
would be nice if we could infer resumption parameters from the outputs already present in the workspace

Reply:
oh I see, the timestring is just the run identifier. nm

Reply:
yeah without giving the resume_timestring it'll do animation in entirely new folder